swagger_django_generator/templates/stubs.py

```diff
@@ -15,7 +15,7 @@ class AbstractStubClass(object):
     {% for verb, info in verbs|dictsort(true) %}
 
     @staticmethod
-    def {{ info.operation }}(request, {% if info.body %}body, {% endif %}{% for ra in info.required_args %}{{ ra.name }}, {% endfor %}{% for oa in info.optional_args %}{{ oa.name }}=None, {% endfor %}*args, **kwargs):
+    def {{ info.operation }}(request, {% if info.body %}body, {% endif %}{% if info.formdata %}formdata, {% endif %}{% for ra in info.required_args %}{{ ra.name }}, {% endfor %}{% for oa in info.optional_args %}{{ oa.name }}=None, {% endfor %}*args, **kwargs):
```

Comment:
I see you use `formdata` here and `formData` elsewhere. It needs to be consistent. Can we maybe change it to `form_data` everywhere, please?